### PR TITLE
Make mkcloud setuphost succeed on s390x

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by aplanas** [comment]
+1

**Comment by toabctl** [comment]
+1

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 # based on https://github.com/SUSE/cloud/wiki/SUSE-Cloud-Installation-Manual
 
-[[ $(uname -m) =~ ^(aarch64|x86_64)$ ]] || { echo "ERROR: need 64bit" ; exit 1 ; }
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: I think, we could even drop this line completely. If someone really finds a i586 VM these days, he will notice a bit later that the repos dont work.

Comment (jdsn) [inline scripts/qa_crowbarsetup.sh]: I also would drop this line entirely.

**Comment by jdsn** [comment]
:-1: besides the line to be dropped

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: it is dropped now.

**Pull request merged**